src/App.vue

```diff
@@ -74,7 +74,7 @@ export default {
     },
 
     createWatermark() {
-      if (this.currentUser?.username && this.publicSettings?.SECURITY_WATERMARK_ENABLED && this.$store.getters.hasValidLicense) {
+      if (this.currentUser?.username && this.publicSettings?.SECURITY_WATERMARK_ENABLED) {
         this.watermark = new Watermark({
           content: this.getWaterMarkContent(),
           width: this.publicSettings?.SECURITY_WATERMARK_WIDTH,
```

Comment:
No, there have been no differences between the code snippet you provided as of December 31, 2021. The watermarks functionality appears to be working correctly with the current version. However, if anything changes, please feel free to let me know!